contracts/mocks/tokens/BadTokenMock.sol

```diff
@@ -31,7 +30,7 @@ contract BadTokenMock is StandardToken {
   */
   function transfer(address _to, uint256 _value) public returns (bool) {
     require(_to != address(0));
-    require(_value <= balances[msg.sender]);
+    require(_value <= balanceOf(msg.sender));
```

Comment:
I wonder if this adds to the call stack by making it a function that just accesses the mapping

Reply:
I assume it does